kvbc/test/categorization/blockchain_test.cpp

```diff
@@ -0,0 +1,271 @@
+// Concord
+//
+// Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the
+// "License").  You may not use this product except in compliance with the
+// Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+
+/**
+ * The following test suite tests ordering of KeyValuePairs
+ */
+
+#include "gtest/gtest.h"
+#include "categorization/column_families.h"
+#include "categorization/updates.h"
+#include "categorization/kv_blockchain.h"
+#include <iostream>
+#include <string>
+#include <utility>
+#include <vector>
+#include <random>
+#include "storage/test/storage_test_common.h"
+
+using concord::storage::rocksdb::NativeClient;
+using namespace concord::kvbc::categorization;
+using namespace concord::kvbc::categorization::detail;
+using namespace concord::kvbc;
+
+namespace {
+
+class categorized_kvbc : public ::testing::Test {
+  void SetUp() override {
+    cleanup();
+    db = TestRocksDb::createNative();
+    db->createColumnFamily(BLOCKS_CF);
+    db->createColumnFamily(ST_CHAIN_CF);
```

Comment:
Maybe the KV blockchain should create the column families (if not existing) on construction?

Reply:
Probably, the constructor now is trivial, I'll start handling it in the following PR I believe.